gtars/src/igd/create.rs

```diff
@@ -100,11 +103,11 @@ pub fn igd_get_create_matches(matches: &ArgMatches) {
         .get_one::<String>("dbname")
         .expect("File list path is required");
 
-    create_igd_f(output_path, filelist, db_output_name);
+    let _igd = create_igd_f(output_path, filelist, db_output_name);
 }
 
 /// Creates IGD database from a directory of bed files.
-pub fn create_igd_f(output_path: &String, filelist: &String, db_output_name: &String) {
+pub fn create_igd_f(output_path: &String, filelist: &String, db_output_name: &String) -> igd_t {
```

Comment:
Is `igd_t` a struct? What do you think about capitalizing it to be more "Rusty"?

Reply:
At some point we can do a large refactor to tackle things like this. For example, some variables are named in a way that are not super clear. This is because they were ported directly from C code and I wanted to keep them aligned for troubleshooting purposes.